src/objects/Data_Import_Settings__c.object

```diff
@@ -117,6 +152,18 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Field_Mapping_Method__c</fullName>
+        <description>Defines the way fields on the NPSP Data Import record are mapped to other objects in Salesforce. The options are Help Text or Data Import Advanced Mapping.</description>
```

Comment:
The actual value is "Data Import Field Mapping" Can we update the description to match that:
 "Defines the way fields on the NPSP Data Import record are mapped to other objects in Salesforce. The options are Help Text or Data Import Field Mapping."